gallery_dl/extractor/twitter.py

```diff
@@ -297,8 +298,11 @@ def _transform_user(self, user):
         except KeyError:
             pass
 
+        # try to fetch extended user data
```

Comment:
Move this into the `elif` block.
This comment doesn't really belong here, does it?